install/ansible/group_vars/dashboard.yml

```diff
@@ -0,0 +1,78 @@
+# Copyright 2022 The SODA Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+# Dummy variable to avoid error because ansible does not recognize the
+# file as a good configuration file when no variable in it.
+dummy:
+
+
+###########
+# GENERAL #
+###########
+
+## install_dashboard: true :- Will install the SODA Dashboard
+## install_dashboard: false :- Will not install the SODA Dashboard
+
+install_dashboard: true
+
+# Dashboard installation types are: 'container', 'source_code'
+dashboard_installation_type: container
+
+# These fields are NOT suggested to be modified
+dashboard_work_dir: /opt/opensds-dashboard-linux-amd64
+
+#Dashboard Console Login Address
+console_login_url: "{{ host_ip }}:8088"
+
+
+##########
+# DOCKER #
+##########
+
+dashboard_docker_image: sodafoundation/dashboard:{{ dashboard_release_version }}
+
+##############
+# SOURCECODE #
+##############
+
+# If user specifies intalling from source code, then he can choose the specific
+# repository branch
+dashboard_repo_branch: "{{ repo_branch }}"
```

Comment:
Should it not come from the SODA installer configurations? Do we need all these project configurations as a part of dashboard? In that case, we need to have project enable / disable support right?!
I am thinking, should we have a simple installer for dashboard and rest handled by SODA installer?!